src/loaders/zarrLoader.js

```diff
@@ -71,23 +91,24 @@ export default class ZarrLoader {
    * @param {Array} loaderSelection, Array of valid dimension selections
    * @returns {Object} data: TypedArray[], width: number, height: number
    */
-  async getRaster({ z, loaderSelection }) {
+  async getRaster({ z, loaderSelection = [] }) {
     const source = this._getSource(z);
     const [xIndex, yIndex] = ['x', 'y'].map(k => this._dimIndices.get(k));
-    const selections = loaderSelection || this._defaultSelection;
```

Comment:
Why get rid of `this._defaultSelection`?

Reply:
I think having a default selection is a hidden side-effect. In order to get image data, a selection should be specified. If someone wants to render a channel, they should need to specify `color`, `slider`, `channelIsOn` and `selection`. We don't use the default selection in practice, and I think it adds unnecessary confusing/complexity.